cypress/utils/constants.js

```diff
@@ -18,10 +18,36 @@ export const BASE_AD_PATH = BASE_PATH + '/app/anomaly-detection-dashboards#';
 
 export const BASE_AD_DASHBOARDS_PATH = BASE_AD_PATH + '/dashboard';
 
+export const BASE_AD_DETECTOR_LIST_PATH = BASE_AD_PATH + '/detectors';
+
+export const BASE_AD_CREATE_AD_PATH = BASE_AD_PATH + '/create-ad';
+
 export const BASE_AD_NODE_API_PATH = BASE_PATH + '/api/anomaly_detectors';
 
 export const AD_GET_DETECTORS_NODE_API_PATH =
   BASE_AD_NODE_API_PATH + '/detectors*';
 
 export const AD_GET_INDICES_NODE_API_PATH =
   BASE_AD_NODE_API_PATH + '/_indices*';
+
+export const AD_GET_MAPPINGS_NODE_API_PATH =
+  BASE_AD_NODE_API_PATH + '/_mappings*';
+
+export function getADStartDetectorNodeApiPath(detectorId) {
+  return BASE_AD_NODE_API_PATH + '/detectors/' + detectorId + '/start';
+}
+
+export function getADStopDetectorNodeApiPath(detectorId) {
+  return BASE_AD_NODE_API_PATH + '/detectors/' + detectorId + '/stop';
+}
+
+export const TEST_DETECTOR_ID = 'ulgqpXEBqtadYz9j2MHG';
+
+// TODO: when repo is onboarded to typescript we can convert these detector states into an enum
```

Comment:
Feel free to create a feature request issue.

Reply:
https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/93